src/Hyperlink/index.jsx

```diff
@@ -59,6 +60,7 @@ Hyperlink.defaultProps = {
   externalLinkTitle: 'Opens in a new window',
   variant: 'default',
   isInline: false,
+  hideLaunchIcon: false,
```

Comment:
If we can make this "icon" with default true. That would make the code a lot cleaner